packages/request-manager/src/httpPool.ts

```diff
@@ -0,0 +1,67 @@
+import http from 'http';
+import { getWeakRandomId } from '@trezor/utils';
+import { InterceptorOptions } from './types';
+
+export class RequestPool {
+    requestPool: any[] = [];
+    timeConsideredTooMuch = 1000 * 10;
+    isNetworkMisbehaving = false;
```

Comment:
What do you think about `isNetworkSlow`?  My idea using Misbehaving here was that this kind of situation could be caused by network being slow but also because some Tor node might be not working well, lost connection .....

So using Misbehaving sounded generic enough to me. :thinking: